Extensions/find_inactives.js

```diff
@@ -283,6 +284,22 @@ XKit.extensions.find_inactives = new Object({
 
 		});
 
+		$(".unfollow_button").bind("click", function() {
```

Comment:
a little confused about why we have our own click handler here. shouldn't tumblr's click handler for unfollow-blog handle this? is there a way to make it handle this? that would be preferable to duplicating the code